test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestFromCsprojForMultipleTFM.cs

```diff
@@ -75,5 +76,23 @@ public void XunitMultiTFM()
             result.StdOut.Should().Contain("Failed   TestNamespace.VSTestXunitTests.VSTestXunitFailTestNetCoreApp");
             result.ExitCode.Should().Be(1);
         }
+
+        [Fact]
+        public void ItCanTestAMultiTFMProjectWithImplicitRestore()
+        {
+            var testInstance = TestAssets.Get(
+                    TestAssetKinds.DesktopTestProjects,
+                    "MultiTFMXunitProject")
+                .CreateInstance()
+                .WithSourceFiles();
+
+            string projectDirectory = Path.Combine(testInstance.Root.FullName, "XUnitProject");
+
+            CommandResult result = new DotnetTestCommand()
+                       .WithWorkingDirectory(projectDirectory)
+                       .ExecuteWithCapturedOutput($"{TestBase.ConsoleLoggerOutputNormal} --framework netcoreapp2.0");
+
+            result.StdOut.Should().Contain("Total tests: 2. Passed: 1. Failed: 1. Skipped: 0.");
```

Comment:
I think I will change this entirely. I will make both tests pass and just check for exit == 0. The other tests above are already looking into the print out. I will save this debt.